edgelet/edgelet-kube/tests/runtime.rs

```diff
@@ -463,6 +526,44 @@ fn create_module_spec(name: &str) -> ModuleSpec<DockerConfig> {
     .unwrap()
 }
 
+fn get_service_account_with_annotations_handler() -> impl Fn(Request<Body>) -> ResponseFuture + Clone
+{
+    move |_| {
+        response(StatusCode::OK, || {
+            json!({
+                "kind": "ServiceAccount",
+                "apiVersion": "v1",
+                "metadata": {
+                    "name": "edgeagent",
+                    "namespace": "my-namespace",
+                    "annotations": {
+                        "net.azure-devices.edge.original-moduleid": "$edgeAgent"
+                    }
+                }
+            })
+            .to_string()
+        })
+    }
+}
+
+fn get_service_account_without_annotations_handler(
+) -> impl Fn(Request<Body>) -> ResponseFuture + Clone {
+    move |_| {
+        println!("adf");
```

Comment:
Was this a testing debug println! that was left by mistake?

Reply:
nice catch!